starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,48 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link href="css/main.css" rel="stylesheet">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+<div class= "header"><h1>Apple Pie</h1></div>  
+
+<div class= "intro"><h2>This was my grandmother's apple pie recipe. I have never seen another one
```

Comment:
instead of creating a div to wrap your h2, you should add a class to your h2 to add some style to it (it's better to keep you code as light as possible :))